### ISSUE TITLE
Update page-specific illustration colors

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#738]
## Overview

We've [revised our color palette](https://v-next--cloudfour-patterns.netlify.app/?path=/docs/design-colors--page) to be more harmonious while taking accessible color contrast into account. These changes have resulted in our previous illustrations becoming a bit out of sync with surrounding UI.

This task involves updating those illustrations so their colors are sourced from the new palette where possible.

The current source files live in our shared Drive in Projects ▸ cloudfour.com ▸ Illustrations ▸ Page Content.

## Outcome

- Old versions of illustrations should be moved to a new "Archive" or "Legacy" folder within the aforementioned "Page Content" folder for backup.
- Illustrator files should be updated with new colors.
- SVG files for these illustrations should be exported and included as assets in our project. (If this is blocked by #725, we can carve this part off as a separate issue.)

### [What We Do](https://cloudfour.com/does/)
- [x] Banner robot
- [x] Responsive web design
- [x] Progressive web apps
- [x] User experience
- [x] Web performance
- [x] Training and workshops

### [Our Approach](https://cloudfour.com/believes/)
- [x] Banner flag person
- [x] Idea-share balloons
- [x] Kanban board

**Comment by tylersticka** [comment]
This has been completed, but the new assets aren't represented outside of one or two components. Will create separate issues to address that.
